src/og/control/DebugInfo.ts

```diff
@@ -136,94 +133,94 @@ class DebugInfo extends Control {
             this.addWatches([
                 {
                     label: "Nodes count",
-                    frame: () => p._renderedNodes.length
+                    frame: () => p?._renderedNodes.length
```

Comment:
Just in case, it's not critical, but interesting question Why do I use `!` instead of` ?`, when '?' is safe? The answer is, that the main idea is that all` xxx!`, is already instantiated, and if it's not (sometimes), so it's a bug and needs attention.

Reply:
...but now, it's fine. In the debugInfo one more ifstatement is not critical.

Reply:
Good point. I fixed it everywhere.